src/mono/mono/mini/simd-intrinsics.c

```diff
@@ -318,6 +320,99 @@ type_to_insert_op (MonoType *type)
 	}
 }
 
+struct IntrinGroup {
+    const char *name;
+    MonoCPUFeatures feature;
+    const SimdIntrinsic *intrinsics;
+    int intrinsics_size;
+    gboolean jit_supported;
+};
+
+struct EmitIntrinsicCtx {
+	gboolean is_64bit;
```

Comment:
Why is this needed ? The arguments have to be packed/unpacked around the call and its only used for 1 function, the emit function.

Reply:
I am guessing the reason why this is needed is because otherwise we will have to pass all those arguments into `emit_hardware_intrinsics` separately. Then, the function signature will be super long.